### PR TITLE
skate: add page

### PR DESCRIPTION
**Pull request by fishsticksnom**
skate
Add skate.md into linux and osx directories.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by fishsticksnom** [comment]
Hi there this is my first contribution, I hope I did it right.

File: pages/linux/skate.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# skate
+
+> A personal key-value store.
+> Skate is simple and powerful. Use it to save and retrieve anything you’d like—even binary data. It’s fully encrypted, backed up to the cloud (that you can self-host if you want) and can be synced with all your machines.
```
Comment (einverne) [inline pages/linux/skate.md:4]: Can we merge these two lines?

```suggestion
> Skate is simple and powerful key-value store. Skate can be used to save and retrieve anything you’d like—even binary data. It’s fully encrypted, backed up to the cloud (that you can self-host if you want) and can be synced with all your machines.
```

File: pages/osx/skate.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# skate
+
+> A personal key-value store.
```
Comment (einverne) [inline pages/osx/skate.md:3]: Same here.

File: pages/linux/skate.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+> More information: <https://github.com/charmbracelet/skate>.
+
+- Store a key and a value on the default database:
+
+`skate set {{"key"}} {{"value"}}`
```
Comment (einverne) [inline pages/linux/skate.md:9]: How about this?

```suggestion
`skate set "{{key}}" "{{value}}"`
```

File: pages/linux/skate.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- Show your keys saved on the default database:
+
+`skate list`
+
+- Delete key and value from the default database:
+
+`skate delete {{"key"}}`
```
Comment (einverne) [inline pages/linux/skate.md:17]:
```suggestion
`skate delete "{{key}}"`
```

File: pages/linux/skate.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+- Create a new key and value in a new database:
+
+`skate set {{"key"}}@{{"database_name"}} {{"value"}}`
```
Comment (einverne) [inline pages/linux/skate.md:21]: same here

File: pages/linux/skate.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+- Show your keys saved in a non default database:
+
+`skate list @{{"database_name"}}`
```
Comment (einverne) [inline pages/linux/skate.md:25]: same here.

File: pages/linux/skate.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+- Delete key and value from a specific database:
+
+`skate delete {{"key"}}@{{"database_name"}}`
```
Comment (einverne) [inline pages/linux/skate.md:29]: same here.

File: pages/linux/skate.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+- Show the databases available:
+
+`skate list-dbs`
+
+- Delete local db and pull down fresh copy from Charm Cloud:
+
+`skate reset @{{"database_name"}}`
```
Comment (einverne) [inline pages/linux/skate.md:37]:
```suggestion
`skate reset @"{{database_name}}"`
```

**Review by einverne — CHANGES_REQUESTED** [review]
Hi @fishsticksnom thank you for your new page. There is a little format suggestion for you to take a look at.

**Comment by fishsticksnom** [comment]
Thank you for fixing it.

File: pages/linux/skate.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+`skate list`
+
+- Delete key and value from the default database:
+
+`skate delete "{{key}}"`
+
+- Create a new key and value in a new database:
+
+`skate set {{"key"}}@{{"database_name"}} {{"value"}}`
```
Comment (kbdharun) [inline pages/linux/skate.md:20]:
```suggestion
`skate set "{{key}}"@"{{database_name}}" "{{value}}"`
```

File: pages/linux/skate.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+- Show your keys saved in a non default database:
+
+`skate list @{{"database_name"}}`
```
Comment (kbdharun) [inline pages/linux/skate.md:24]:
```suggestion
`skate list @"{{database_name}}"`
```

File: pages/linux/skate.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+- Delete key and value from a specific database:
+
+`skate delete {{"key"}}@{{"database_name"}}`
```
Comment (kbdharun) [inline pages/linux/skate.md:28]:
```suggestion
`skate delete "{{key}}"@"{{database_name}}"`
```

File: pages/osx/skate.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+> Skate is simple and powerful. Use it to save and retrieve anything you’d like—even binary data. It’s fully encrypted, backed up to the cloud (that you can self-host if you want) and can be synced with all your machines.
+> More information: <https://github.com/charmbracelet/skate>.
+
+- Store a key and a value on the default database:
+
+`skate set {{"key"}} {{"value"}}`
```
Comment (kbdharun) [inline pages/osx/skate.md:9]:
```suggestion
`skate set "{{key}}" "{{value}}"`
```

File: pages/osx/skate.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+
+- Show your keys saved on the default database:
+
+`skate list`
+
+- Delete key and value from the default database:
+
+`skate delete {{"key"}}`
```
Comment (kbdharun) [inline pages/osx/skate.md:17]:
```suggestion
`skate delete "{{key}}"`
```

File: pages/osx/skate.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+
+- Create a new key and value in a new database:
+
+`skate set {{"key"}}@{{"database_name"}} {{"value"}}`
```
Comment (kbdharun) [inline pages/osx/skate.md:21]:
```suggestion
`skate set "{{key}}"@"{{database_name}}" "{{value}}"`
```

File: pages/osx/skate.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+
+- Show your keys saved in a non default database:
+
+`skate list @{{"database_name"}}`
```
Comment (kbdharun) [inline pages/osx/skate.md:25]:
```suggestion
`skate list @"{{database_name}}"`
```

File: pages/osx/skate.md [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+
+- Delete key and value from a specific database:
+
+`skate delete {{"key"}}@{{"database_name"}}`
```
Comment (kbdharun) [inline pages/osx/skate.md:29]:
```suggestion
`skate delete "{{key}}"@"{{database_name}}"`
```

File: pages/osx/skate.md [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+
+- Show the databases available:
+
+`skate list-dbs`
+
+- Delete local db and pull down fresh copy from Charm Cloud:
+
+`skate reset @{{"database_name"}}`
```
Comment (kbdharun) [inline pages/osx/skate.md:37]:
```suggestion
`skate reset @"{{database_name}}"`
```

File: pages/linux/skate.md [hunk 18]
```diff
@@ -0,0 +1,36 @@
+# skate
+
+> Skate is simple and powerful key-value store. Skate can be used to save and retrieve anything you’d like—even binary data. It’s fully encrypted, backed up to the cloud (that you can self-host if you want) and can be synced with all your machines.
```
Comment (kbdharun) [inline pages/linux/skate.md:3]:
```suggestion
> Simple and powerful key-value store.
```
I think this would be better

File: pages/osx/skate.md [hunk 19]
```diff
@@ -0,0 +1,37 @@
+# skate
+
+> A personal key-value store.
+> Skate is simple and powerful. Use it to save and retrieve anything you’d like—even binary data. It’s fully encrypted, backed up to the cloud (that you can self-host if you want) and can be synced with all your machines.
```
Comment (kbdharun) [inline pages/osx/skate.md:4]:
```suggestion
> Simple and powerful key-value store.
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Tip:** You can batch all suggestions in the Files tab on the web interface and make a single commit.

File: pages/linux/skate.md (hunk 18 above)
Comment (Magrid0) [reply pages/linux/skate.md:3]: yes, for the long description there's always `man`

**Review by kbdharun — COMMENTED** [review]
Since `skate` is available on multiple platforms with the same syntax, I suggest having this page in the `common` directory instead of  `linux` and `osx` directories as it make it easy to translate for translators. After this the PR is GTG, Thanks for your contribution.

**Comment by fishsticksnom** [comment]
> Since `skate` is available on multiple platforms with the same syntax, I suggest having this page in the `common` directory instead of `linux` and `osx` directories as it make it easy to translate for translators. After this the PR is GTG, Thanks for your contribution.

Absolutely! 
Thank you for helping me with my first pull request.

I have a questions should I close this pull request, fix the typos, move skate.md to `common` and then open make a new pull request?

File: pages/common/skate.md [hunk 20]
```diff
@@ -0,0 +1,37 @@
+# skate
+
+> A personal key-value store.
```
Comment (kbdharun) [inline pages/common/skate.md:3]:
```suggestion
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by einverne — APPROVED** [review]
Looks good to me.

**Pull request merged**